lib/galaxy/web/base/interactive_environments.py

```diff
@@ -69,6 +71,22 @@ def __init__(self, trans, plugin):
         else:
             self.attr.proxy_prefix = ''
 
+    def load_allowed_images(self):
+        if os.path.exists(os.path.join(self.attr.our_config_dir, 'allowed_images.yml')):
+            fn = os.path.join(self.attr.our_config_dir, 'allowed_images.yml')
+        elif os.path.exists(os.path.join(self.attr.our_config_dir, 'allowed_images.yml.sample')):
+            fn = os.path.join(self.attr.our_config_dir, 'allowed_images.yml.sample')
+        else:
+            raise Exception("Could not find allowed_images.yml file for " + self.attr.viz_id)
```

Comment:
Yes, absolutely. That was the original idea, I'm not, for the life of me, understanding why I raised an exception here. I definitely shouldn't be.